.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# These owners will be the default owners for everything in the repo.
+*       @ryanzhang-oss @ealianis
```

Comment:
Can you move each person to their own line? That's kinda the convention and I am not sure if the current format works.

Reply:
Good point. I was referencing what I saw on their docs. Let's give it a try.